starboard/linux/shared/cobalt/configuration.py

```diff
@@ -104,5 +111,8 @@ def GetTestEnvVariables(self):
         },
         'net_unittests': {
             'ASAN_OPTIONS': 'detect_leaks=0'
+        },
+        'starboard_platform_tests': {
+            'ASAN_OPTIONS': 'detect_odr_violation=0'
```

Comment:
Do you want to add TODO for this one and the one above , so that people can start investigating where you left of.

Reply:
Done